src/main/scala/beam/agentsim/infrastructure/ChargingNetwork.scala

```diff
@@ -235,23 +257,35 @@ object ChargingNetwork {
   }
 
   final case class ChargingStation(zone: ParkingZone[_]) {
-    import ConnectionStatus._
-    private val connectedVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
+    import ChargingStatus._
+    private val chargingVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
 
-    private val waitingLineInternal: mutable.PriorityQueue[ChargingVehicle] =
+    private val gracedVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
```

Comment:
Is this an industry term? Otherwise it doesn't make itself immediately evidenced as far as I can tell.